### ISSUE TITLE
Zend_Date::setTime buggy - adds +1h on a date when DST is changed.

### ISSUE THREAD

**Issue by mh-ovos** [zendframework/zf1#682]
Hello,
This is a critical bug, it turned our system upside down today.
This has been reported long time ago, so I am not going to rewrite everything again and again here.
Please see:
http://framework.zend.com/issues/browse/ZF-10584
http://stackoverflow.com/questions/7181702/work-around-for-zend-date-dst-bug

Since it's not easy to override Zend_Date (would need to change all Zend_Date occurrences in custom codebase or use custom fork of zf1 repo) I beg you, fix this in the main repo.

Happy Easter, to those who have to work today:P

**Comment by flipace** [comment]
+1

**Comment by froschdesign** [comment]
Yes, this is a bug. I will check the workaround, but there is not much light at end of tunnel…
